package.json

```diff
@@ -40,7 +41,7 @@
     "build:full": "npm run clean:lerna && npm run bootstrap && npm test",
     "pretest": "npm run clean && npm run build",
     "test": "node packages/build/bin/run-nyc npm run mocha --scripts-prepend-node-path",
-    "mocha": "node packages/build/bin/run-mocha \"packages/*/DIST/test/**/*.js\" \"examples/*/DIST/test/**/*.js\" \"packages/cli/test/*.js\"",
+    "mocha": "node packages/build/bin/run-mocha \"packages/*/DIST/test/**/*.js\" \"examples/*/DIST/test/**/*.js\" \"packages/cli/test/*.js\" \"packages/build/test/integration/*.js\"",
```

Comment:
Would you mind making this more future-proof by using `\"packages/build/test/*/*.js\"`? I.e. allow any test category, not only `integration`. Unless I am missing something and the wildcard won't work.

Reply:
I wanted to exclude `packages/build/test/fixtures`. We can move `fixtures` to be inside `integration`.